### PR TITLE
fix: RTL Table styling fixes

### PR DESCRIPTION
**Pull request by kaydbwithers**
## Why
This PR adds styling fixes to the Table when RTL dir is active.

## What
These changes should not affect anything in LTR, they are just fixes for RTL.

**Before**
LTR
image

RTL
In RTL the tooltip icon appeared too close to the text.
image

In RTL when a `TableHeaderRowCell` had an alignment of `center` or `end` then the sorting chevron did not appear correctly.
image

**After**
LTR
image

RTL
image
image

### CODE REVIEW

File: draft-packages/table/KaizenDraft/Table/Table.module.scss [hunk 1]
```diff
@@ -101,7 +101,13 @@ $row-height-data-variant: 48px;
 
 .headerRowCellTooltipIcon {
   color: $color-blue-500;
-  margin-right: $spacing-xs;
+
+  [dir="ltr"] & {
+    margin-right: $spacing-xs;
```
Comment (christian314159) [inline draft-packages/table/KaizenDraft/Table/Table.module.scss:106]: Btw, as we're not supporting IE11 anymore, you should be able to use `margin-inline/block` here instead 😃

Comment (kaydbwithers) [reply draft-packages/table/KaizenDraft/Table/Table.module.scss]: Thanks @christian314159, I've updated it.

File: .changeset/smooth-dogs-cheat.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 "@kaizen/draft-table": patch
 ---
 
-Align the tooltip to the center of the Table header text
+* Align the tooltip to the center of the Table header text
```
Comment (gyfchong) [inline .changeset/smooth-dogs-cheat.md:5]: Different PR?

Comment (kaydbwithers) [reply .changeset/smooth-dogs-cheat.md]: Okay, it's gone now.

**Comment by christian314159** [comment]
Hey @kaydbwithers I just had a look at the branch Storybook and found this small issue where the tickbox in the header sits too close to the description:
image
I checked and this is an existing issue, not caused by your PR, but I wonder if we could incorporate a fix as well while we're touching the table headings? Feel free to push back.

**Comment by kaydbwithers** [comment]
@christian314159 sure, I've also updated the checkbox margin.

**Review by christian314159 — APPROVED** [review]
🚢 Thanks for contributing!

**Pull request merged**